.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "docs"]
 	path = docs
 	url = https://github.com/pine64/bl602-docs.git
```

Comment:
Maybe add a newline after this, for nicer formatting?

Reply:
@Avamander I can, it was just auto-generated by git?

Reply:
I should probably add that my point is that git generated and maintained files should likely not be edited... It could cause some formatting weirdness down the line (like auto-removing spaces or leaving spaces where they shouldn't be).

Reply:
I haven't seen git do that on any submodule files, I am pretty sure it just appends-deletes the appropriate entries when told to, not more.